src/hssm/distribution_utils/dist.py

```diff
@@ -114,16 +114,22 @@ class SSMRandomVariable(RandomVariable):
         """WFPT random variable."""
 
         name: str = "SSM_RV"
-
-        # NOTE: This is wrong at the moment, but necessary
         # to get around the support checking in PyMC that would result in error
-        ndim_supp: int = 0
+        ndim_supp: int = 1
 
         ndims_params: list[int] = [0 for _ in list_params]
         dtype: str = "floatX"
         _print_name: tuple[str, str] = ("SSM", "\\operatorname{SSM}")
         _list_params = list_params
 
+        # PyTensor, as of version 2.12, enforces a check to ensure that
+        # at least one parameter has the same ndims as the support.
+        # This overrides that check and ensures that the dimension checks are correct.
+        # For more information, see this issue
+        # https://github.com/lnccbrown/HSSM/issues/36
+        def _supp_shape_from_params(*args, **kwargs):
```

Comment:
I guess this is just the diff to main, and includes multiple pr's ?.
Because this change doesn't seem related to the current pr either.

Reply:
This is from a different PR for sure